packages/block-library/src/post-featured-image/dimension-controls.js

```diff
@@ -60,55 +60,45 @@ const DimensionControls = ( {
 	const scaleLabel = _x( 'Scale', 'Image scaling options' );
 	return (
 		<PanelBody title={ __( 'Dimensions' ) }>
-			<Flex justify="space-between">
-				<FlexItem>
-					<UnitControl
-						label={ __( 'Height' ) }
-						labelPosition="top"
-						value={ height || '' }
-						onChange={ ( nextHeight ) => {
-							onDimensionChange( 'height', nextHeight );
-						} }
-						units={ units }
-					/>
-				</FlexItem>
-				<FlexItem>
-					<UnitControl
-						label={ __( 'Width' ) }
-						labelPosition="top"
-						value={ width || '' }
-						onChange={ ( nextWidth ) => {
-							onDimensionChange( 'width', nextWidth );
-						} }
-						units={ units }
-					/>
-				</FlexItem>
-			</Flex>
-			{ !! height && (
-				<>
-					<BaseControl
-						aria-label={ scaleLabel }
-						className="block-library-post-featured-image-scale-controls"
-					>
-						<div>
-							<BaseControl.VisualLabel>
-								{ scaleLabel }
-							</BaseControl.VisualLabel>
-						</div>
-						<SegmentedControl
-							label={ scaleLabel }
-							value={ scale }
-							onChange={ ( value ) => {
-								setAttributes( {
-									scale: value,
-								} );
+			<BaseControl>
+				<Flex justify="space-between">
+					<FlexItem>
+						<UnitControl
+							label={ __( 'Height' ) }
+							labelPosition="top"
+							value={ height || '' }
+							onChange={ ( nextHeight ) => {
+								onDimensionChange( 'height', nextHeight );
+							} }
+							units={ units }
+						/>
+					</FlexItem>
+					<FlexItem>
+						<UnitControl
+							label={ __( 'Width' ) }
+							labelPosition="top"
+							value={ width || '' }
+							onChange={ ( nextWidth ) => {
+								onDimensionChange( 'width', nextWidth );
 							} }
-							isBlock
-						>
-							{ SCALE_OPTIONS }
-						</SegmentedControl>
-					</BaseControl>
-				</>
+							units={ units }
+						/>
+					</FlexItem>
+				</Flex>
+			</BaseControl>
```

Comment:
Why there's BaseControl here?

Reply:
Because `Flex` hasn't the `BaseControl` inside.

Reply:
I'm not sure I follow, `Flex` is about design and `BaseControl` is about semantics, it adds a label and potentially help to a single control and it seems here we already have two controls `UnitControl` with their labels... so it seems there's no need for a `BaseControl`

Reply:
I have the same thoughts as @youknowriad — `UnitControl` seems to already have labels, which means that `BaseControl` should not be needed there

Reply:
I removed it and added css for displaying as the rest controls used in Inspector toolbar here: https://github.com/WordPress/gutenberg/pull/34017/commits/70073cf5f8ee5998224812c10fede111cdc02a8c